libs/api/domains/national-registry/src/lib/graphql/user.resolver.ts

```diff
@@ -28,7 +28,7 @@ export class UserResolver {
     return this.nationalRegistryService.getUser(user.nationalId)
   }
 
-  @ResolveField('citizenship', () => String)
+  @ResolveField('citizenship', () => Citizenship)
   resolveCitizenship(@Parent() user: User): string {
     return user.citizenship.name
```

Comment:
Need to change the resolver as well. Shouldn't just return the name

Reply:
Got it! Updated :)